src/Resources/config/services.yml

```diff
@@ -0,0 +1,7 @@
+parameters:
+    property_accessor.class: Symfony\Component\PropertyAccess\PropertyAccessor
```

Comment:
parameters are shared across all installed bundles, I think you should prefix its name with the bundle name to avoid collisions